ivadomed/utils.py

```diff
@@ -61,7 +62,16 @@ def __init__(self, fname_pred, fname_gt, params={}):
         self.fname_gt = fname_gt
 
         self.data_pred = self.get_data(self.fname_pred)
-        self.data_gt = self.get_data(self.fname_gt)
+
+        h, w, d = self.data_pred.shape[:3]
+        self.n_classes = len(self.fname_gt)
+        self.data_gt = np.zeros((h, w, d, self.n_classes))
+
+        for idx, file in enumerate(fname_gt):
+            if os.path.exists(file):
+                self.data_gt[..., idx] = self.get_data(file)
+            else:
+                self.data_gt[..., idx] = np.zeros((h, w, d), dtype='u1')
```

Comment:
In which case(s) are we falling here?

Reply:
In case, the path doesn't exist. For instance, not all my subject has edema and cavity. If does paths don't exist for a specific user, the gt should be an empty array